frontend/src/screens/learner.tsx

```diff
@@ -201,45 +200,93 @@ export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }
+    }
+
+    const [hoveredLearningPath, setHoveredLearningPath] = useState<string | null>(null)
+
+    const handleMouseEnter = (learningPath: string) => {
+        setHoveredLearningPath(learningPath)
+    }
+
+    const handleMouseLeave = () => {
+        setHoveredLearningPath(null)
+    }
+
     return (
-        <Stack gap='lg' data-testid='studies-listing'>
-            {learningPaths.map(learningPath => {
-                if (!learningPath) return null
-                const studies = sortBy(studiesByLearningPath[learningPath.label], (study) => !!study.completedAt)
-
-                return (
-                    <Stack key={learningPath.label}>
-                        <Group gap='sm'>
-                            <Title order={3}>
-                                {learningPath.label}
-                            </Title>
-                            <Text span>|</Text>
-                            <Title order={3} fw='300'>
-                                {learningPath.description}
-                            </Title>
-                            {learningPath.completed ? <Badge c={colors.text} color={colors.green}>Completed</Badge> : null}
+        <Container style={{ display: 'flex', flexDirection: 'row', width: '100%' }}>
+            <Box style={{ width: '25%' }}>
+                {learningPaths.map(learningPath => {
+                    if (!learningPath) return null
+                    return (
+                        <Group 
+                            key={learningPath.label}
+                            style={{ cursor: 'pointer', 
+                                color: hoveredLearningPath === learningPath.label ? colors.blue : colors.gray70,
+                                padding: '0.5rem',
+                            }}
+                            onClick={() => scrollToLearningPath(learningPath.label)}
+                            onMouseEnter={() => handleMouseEnter(learningPath.label)}
+                            onMouseLeave={handleMouseLeave}
+                        >
+                            <Text>{learningPath.label}</Text>
+                            <Text>
+                                {completedStudiesByLearningPath[learningPath.label]? completedStudiesByLearningPath[learningPath.label].length : 0}
+                                /
+                                {studiesByLearningPath[learningPath.label].length}
+                            </Text>
                         </Group>
-                        {isMobile ?
-                            <MobileStudyCards studies={studies} /> :
-                            <DesktopStudyCards studies={studies} />
-                        }
-                    </Stack>
-                )
-            })}
-        </Stack>
+                    )
+                })}
+            </Box>
+            <Box style={{ width: '75%', gap: 'lg' }} data-testid='studies-listing'>
```

Comment:
@Coder-Srinivas  When possible we should try to use <Stack> or <Group>, or in really custom scenarios <Flex> instead of box as it will have a lot of the flex properties available as props (ie `gap` in this instance) and they [can take shorthand CSS properties](https://mantine.dev/styles/style-props/)

Reply:
Got it.